apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -40,6 +40,13 @@ import { HttpErrorResponse } from '@angular/common/http';
 import { DotPropertiesService } from '@services/dot-properties/dot-properties.service';
 import { DotLicenseService } from '@services/dot-license/dot-license.service';
 
+interface DotIframeEditEvent {
+    container?: DOMStringMap;
+    dataset?: DOMStringMap;
+    data?: { [key: string]: DOMStringMap };
```

Comment:
Are you 100% sure about this?

Reply:
![image](https://user-images.githubusercontent.com/37185433/154570987-f2e29d1e-bd01-471d-a289-fbc9b84c2285.png)

Reply:
in this case we know the type of the attributes are string so `DOMStringMap` works here.